ci-operator/step-registry/openstack/provision/etcd-on-local-disk/openstack-provision-etcd-on-local-disk-commands.sh

```diff
@@ -157,78 +188,220 @@ spec:
         name: var-lib-etcd.mount
       - contents: |
           [Unit]
-          Description=Sync etcd data if new mount is empty
+          Description=Create local-etcd filesystem
           DefaultDependencies=no
-          After=var-lib-etcd.mount var.mount
-          Before=crio.service
+          After=local-fs-pre.target
+
+          # Don't run if the filesystem already exists
+          ConditionPathIsSymbolicLink=!/dev/disk/by-label/local-etcd
```

Comment:
I think that openstack should provide the ephemeral disk without format, right? however it looks the simbolic link is already there.

Reply:
It looks like this is working as expected. The job is executing and migrating content across. Once that content is migrated, it should not run again. It is attempting to do so after reboot but since it already ran in the past (and the device exists), it exits.

This is one downside of doing all this stuff as systemd units, as discussed with @mdbooth, but it's a small downside fortunately.

Reply:
Understood. Thanks for checking this.